mathjax3-ts/output/chtml/fonts/tex.ts

```diff
@@ -414,10 +415,11 @@ export class TeXFont extends FontData {
         }
         if (He) {
             css['border-bottom-width'] = this.em0(He - .03);
-            css['margin-bottom'] = this.em(-He);
+            styles['.MJX-TEX mjx-stretchy-v[c="' + c + '"] > mjx-end'] = {'margin-top': this.em(-He)};
             if (mid) {
-                styles['.MJX-TEX mjx-stretchy-v[c="' + c + '"] > mjx-ext:last-of-type'] = {
-                    'margin-top': this.em(-He)
+                styles['.MJX-TEX mjx-stretchy-v[c="' + c + '"] > mjx-mid'] = {
+                    'margin-top': this.em(-Hm/2),
+                    'margin-bottom': this.em(-Hm/2)
```

Comment:
Why can't the ``mid`` case from lines 407-411 be handled here?
Can we ever get a mid part without end part?
Or is it important that the styles is set earlier?

Reply:
Not sure why it is in the location.  Probably due to doing one part at one time and the other at later time.  I've moved them to the same location and tidied up a bit.

Reply:
Oh, I see why it was the way.  The original CSS was for a different element, but was changed to the mid element, and I didn't catch that it could be moved earlier.